### ISSUE TITLE
Approach vector pointing out of the object

### ISSUE THREAD

**Issue by 1mingW** [graspnet/graspnet-baseline#77]
Hi,

How did you filter out the invalid approaching vector that pointing out of the object? I got some grasps that try to grasp the cracker box from inside (see in the figure). I am not sure if you already filtered out those views while annotation. Do these views automatically get low view scores in the ApproachNet or we should manually filter them out? 
In addition, I am just using the pretrained weights with 18 epochs that is provided in the repo for this result. Maybe I should train the network longer for better results? Is your demo result also produced with these weights or with weights trained more than 18 epochs? 

![image](https://github.com/graspnet/graspnet-baseline/assets/111709640/687d5685-b108-49fa-917f-dc49b685aee6)

**Comment by chenxi-wang** [comment]
The proportion of such candidates is small and they usually get low scores. If you care about the results, you can filter them according to your requirements, such as vector direction, workspace and more precise collision detection.

The training epoch is just set empirically. You can have a try but I do not think the results will change a lot.

**Comment by abhishek47kashyap** [comment]
> training epoch is just set empirically. You can have a try but I do not think the results will change a lot.

The paper states:
_"The learning rate is decreased to 0.0001 after 60 epochs and then decreased to 0.00001 after 100 epochs."_
![Screenshot from 2024-09-11 19-33-07](https://github.com/user-attachments/assets/b44521c3-8f9b-43d5-ad26-0dc71c63756f)


So if the provided weights are from training for 18 epochs, then the learning rate never changed from `0.001`.
